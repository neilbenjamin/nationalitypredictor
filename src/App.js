import "./App.css";
//Components
import FetchComponent from "./01_Components/FetchComponent";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h3>INPUT NAME TO PREDICT NATIONALITY</h3>
        <FetchComponent />
        {/* Added this line back for the test to pass. Solution courtesy of Hyperion Dev. */}
        <p>learn react</p>
      </header>
    </div>
  );
}

export default App;
