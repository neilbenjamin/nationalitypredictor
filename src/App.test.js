import { render } from "@testing-library/react";
import App from "./App";

// test("renders learn react link", () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

test("renders App snapshot", () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
