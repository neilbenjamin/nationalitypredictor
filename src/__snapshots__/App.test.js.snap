// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <h3>
        INPUT NAME TO PREDICT NATIONALITY
      </h3>
      <div>
        <input
          value=""
        />
        <br />
        <button>
          SUBMIT
        </button>
        <div />
      </div>
      <p>
        learn react
      </p>
    </header>
  </div>
</DocumentFragment>
`;
